### PR TITLE
Add type for Sankey chart for plotly.js

### PR DESCRIPTION
**Pull request by jpabdou**
* Add Sankey type to plotly.js

* Add test with Sankey chart

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://plotly.com/javascript/reference/sankey/#sankey
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## plotly.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/plotly.js@latest/))
was missing the following properties:
1. version
1. Fx
1. Snapshot
1. PlotSchema
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 35903ab59eef6f2f060046aea5080a6ac4039669
</p>

File: types/plotly.js/lib/traces/sankey.d.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        color: SankeyColor;
+        width: number;
+    }>;
+    source: number[];
+    target: number[];
+    value: number[];
+}
+
+export interface SankeyData {
+    type: string;
```
Comment (peterblazejewicz) [inline types/plotly.js/lib/traces/sankey.d.ts:80]: should this read:
```ts
```suggestion
    type: 'sankey';
```

**Review by peterblazejewicz — APPROVED** [review]
@jpabdou thanks!
LGTM!

**Comment by jpabdou** [comment]
Ready to merge

**Pull request merged**